templates/rabbitMQ/rabbitmq.cfg

```diff
@@ -15,122 +15,110 @@ PATROL_CONFIG
-"/SENTRY/X/rabbitmq:channels/credentials" = { REPLACE = "RabbitAPI" },
+"/SENTRY/X/rabbitmq:channels/credentials" = { REPLACE = "System Credentials" },
 "/SENTRY/X/rabbitmq:channels/headers" = { REPLACE = "" },
 "/SENTRY/X/rabbitmq:channels/httpMethod" = { REPLACE = "GET" },
 "/SENTRY/X/rabbitmq:channels/httpRequestContent" = { REPLACE = "textOnly" },
 "/SENTRY/X/rabbitmq:channels/label" = { REPLACE = "Channels" },
 "/SENTRY/X/rabbitmq:channels/messageList" = { REPLACE = "" },
-"/SENTRY/X/rabbitmq:channels/parameterList" = { REPLACE = "ExecutionTime;1,HTTPStatusCode;0,Result;1,Status;1" },
+"/SENTRY/X/rabbitmq:channels/parameterList" = { REPLACE = "ExecutionTime;1,HTTPStatus;1,HTTPStatusCode;0,Result;1,Status;1" },
 "/SENTRY/X/rabbitmq:channels/parent" = { REPLACE = "rabbitmq" },
 "/SENTRY/X/rabbitmq:channels/pollingIntervalList" = { REPLACE = "httpRequestColl;continuous;120;" },
-"/SENTRY/X/rabbitmq:channels/productVersion" = { REPLACE = "10000" },
+"/SENTRY/X/rabbitmq:channels/productVersion" = { REPLACE = "1000" },
 "/SENTRY/X/rabbitmq:channels/reportAsCollectionError" = { REPLACE = "0" },
-"/SENTRY/X/rabbitmq:channels/successfulHttpStatusCode" = { REPLACE = "200-208,226" },
-"/SENTRY/X/rabbitmq:channels/thresholdList" = { REPLACE = "ExecutionTime;0 0 0 0 0 0;1 15 30 0 0 1;1 30 2147483646 0 0 2,Status;0 0 0 0 0 0;1 1 1 0 0 1;1 2 2 0 0 2" },
+"/SENTRY/X/rabbitmq:channels/thresholdList" = { REPLACE = "ExecutionTime;0 0 0 0 0 0;1 15 30 0 0 1;1 30 2147483646 0 0 2,HTTPStatus;0 0 0 0 0 0;1 1 1 0 0 1;1 2 2 0 0 2,Status;0 0 0 0 0 0;1 1 1 0 0 1;1 2 2 0 0 2" },
 "/SENTRY/X/rabbitmq:channels/timeout" = { REPLACE = "30" },
 "/SENTRY/X/rabbitmq:channels/type" = { REPLACE = "HTTPRequest" },
 "/SENTRY/X/rabbitmq:channels/url" = { REPLACE = "http://%{HOSTNAME}:%{PORT}/api/channels" },
 "/SENTRY/X/rabbitmq:channels/variables" = { REPLACE = "" },"/SENTRY/X/rabbitmq:connections/actionList" = { REPLACE = "" },
-"/SENTRY/X/rabbitmq:connections/authenticationType" = { REPLACE = "basic" },
 "/SENTRY/X/rabbitmq:connections/bodyType" = { REPLACE = "none" },
 "/SENTRY/X/rabbitmq:connections/contentType" = { REPLACE = "application/x-www-form-urlencoded" },
-"/SENTRY/X/rabbitmq:connections/credentials" = { REPLACE = "RabbitAPI" },
+"/SENTRY/X/rabbitmq:connections/credentials" = { REPLACE = "System Credentials" },
 "/SENTRY/X/rabbitmq:connections/headers" = { REPLACE = "" },
 "/SENTRY/X/rabbitmq:connections/httpMethod" = { REPLACE = "GET" },
 "/SENTRY/X/rabbitmq:connections/httpRequestContent" = { REPLACE = "textOnly" },
 "/SENTRY/X/rabbitmq:connections/label" = { REPLACE = "Connections" },
 "/SENTRY/X/rabbitmq:connections/messageList" = { REPLACE = "" },
-"/SENTRY/X/rabbitmq:connections/parameterList" = { REPLACE = "ExecutionTime;1,HTTPStatusCode;0,Result;1,Status;1" },
+"/SENTRY/X/rabbitmq:connections/parameterList" = { REPLACE = "ExecutionTime;1,HTTPStatus;1,HTTPStatusCode;0,Result;1,Status;1" },
```

Comment:
There is no HTTPStatus parameter in version 10. I suspect you are using a non-GA version of the product here!